doc/intro-coding

```diff
@@ -0,0 +1,61 @@
+How to contribute code to LedgerSMB.
```

Comment:
Wondering if we should add this to the website instead; we have a community guide which explains some (most?) of what I put into my e-mail: https://ledgersmb.org/community-guide

It's probably a good idea to add information like that to the README.md file for people to find...